atlas/getting-started/deploy-from-existing-repo.mdx

```diff
@@ -25,7 +23,12 @@ For example, the `package.json` file for a Faust.js or Next.js app would need to
   },
 ```
 
-If you don't know what command needs to be run to build your app, please refer to your JavaScript framework's documentation.
+3. creating a "ATLAS_BUILD_SCRIPT" environmental variable in the user-portal and adding the command as value. Learn more about environmental variables [here](#env-var).
+
+![ATLAS_BUILD_SCRIPT env variable](./assets/img/env-variables.png)
+
+
+>If you don't know what command needs to be run to build your app, please refer to the JavaScript framework's documentation.
```

Comment:
@dgallardox I think we should say "your JavaScript framework's documentation" instead of "the JavaScript framework's documentation" here.